src/lib/Libpython/pbs_python_svr_internal.c

```diff
@@ -5831,6 +5847,7 @@ _pbs_python_event_to_request(unsigned int hook_event, hook_output_param_t *req_p
 				&((struct rq_queuejob *)(req_params->rq_job))->rq_attr, NULL, 0) == -1) {
 				return -1;
 			}
+			print_svrattrl_list("pbs_populate_svrattrl_from_python_class==>",  &((struct rq_queuejob *)(req_params->rq_job))->rq_attr);
```

Comment:
I'm looking at as printing the value populated by pbs_populate_svrattrl_from_python_class() that's why I've kept the string as is.